src/switch_core_media.c

```diff
@@ -4733,12 +4733,17 @@ static switch_status_t check_ice(switch_media_handle_t *smh, switch_media_type_t
 
 #ifdef RTCP_MUX
 			} else if (!strcasecmp(attr->a_name, "rtcp-mux")) {
-				engine->rtcp_mux = SWITCH_TRUE;
-				engine->remote_rtcp_port = engine->cur_payload_map->remote_sdp_port;
-				got_rtcp_mux++;
+				const char *use_rtcp_mux = NULL;
+				use_rtcp_mux = switch_channel_get_variable(smh->session->channel, "rtcp_mux");
 
-				if (!smh->mparams->rtcp_audio_interval_msec) {
-					smh->mparams->rtcp_audio_interval_msec = SWITCH_RTCP_AUDIO_INTERVAL_MSEC;
+				if (!use_rtcp_mux || switch_true(use_rtcp_mux)) {
+					engine->rtcp_mux = SWITCH_TRUE;
+					engine->remote_rtcp_port = engine->cur_payload_map->remote_sdp_port;
+					got_rtcp_mux++;
+
+					if (!smh->mparams->rtcp_audio_interval_msec) {
```

Comment:
This should be outside of this if statement. If FS enforce not to support rtcp-mux then it will fallback to rtcp port pair or if rtcp port is provided in the SDP. This require us to set the rtcp interval otherwise the rtcp will be disabled